sdk/storage/azure-storage-blob/tests/test_blob_samples_authentication.py

```diff
@@ -125,3 +125,21 @@ def test_auth_shared_access_signature(self):
         # [END create_sas_token]
         assert sas_token is not None
 
+    def test_auth_default_azure_credential(self):
+        # [START create_blob_service_client_oauth]
+        # Get a credential for authentication
+        # Default Azure Credentials attempt a chained set of authentication methods, per documentation here: https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/identity/azure-identity
+        # For example user (who must be an Azure Event Hubs Data Owner role) to be logged in can be specified by the environment variable AZURE_USERNAME
+        # Alternately, one can specify the AZURE_TENANT_ID, AZURE_CLIENT_ID, and AZURE_CLIENT_SECRET to use the EnvironmentCredentialClass.
+        # The docs above specify all mechanisms which the defaultCredential internally support.
+        from azure.identity import DefaultAzureCredential
+        default_credential = DefaultAzureCredential()
+ 
+        # Instantiate a BlobServiceClient using a token credential
+        from azure.storage.blob import BlobServiceClient
+        blob_service_client = BlobServiceClient(account_url=self.oauth_url, credential=default_credential)
+        # [END create_blob_service_client_oauth]
+ 
+        # Get account information for the Blob Service
+        account_info = blob_service_client.get_service_properties()
+        assert account_info is not None
```

Comment:
1) remove the assert Statement
2) rename to def auth_* insteard of def test_auth_*

Just wanted to make it consistent with https://github.com/Azure/azure-sdk-for-python/pull/8171

Reply:
Shouldn't things also move from `tests` to `samples` in that case?

Reply:
++ Yep, will also be doing that @bryevdv